public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <!-- <link rel="icon" href="%PUBLIC_URL%/favicon.ico" /> -->
-    <link rel="icon" href="./assets/images/logo.png" />
+    <link rel="icon" href="./logo.png" />
```

Comment:
Could we use a const for image paths ?
```suggestion
    <link rel="icon" href={LOGO} />
```